Tasks/Barkun/Task_7/data.py

```diff
@@ -0,0 +1,55 @@
+# key - Name
+# 0: Category, 1: Product Code, 2: Price USD,  3: Stock balance
+
+import json
+
+
+data_json = """{
+    "Glenmorangie" : [
+        "Whisky",
+        "whikey01",
+        245,
+        3],
+    "Johnnie Walker Blue" : [
+        "Whisky",
+        "whikey02",
+        482,
+        5],
+    "Martell XO": [
+        "Cognac",
+        "cog01",
+        225,
+        4],
+    "Napoleon VSOP" : [ 
+        "Cognac",
+        "cog02",
+        30,
+        6],
+    "770 Miles Zinfandel": [
+        "Wine",
+        "wine01",
+        15.30,
+        10],
+    "King Saint Chinian 2020" : [
+        "Wine",
+        "wine02",
+        16.12,
+        20]
+}""" 
```

Comment:
json is a good choice, but it's kinda underused here, you could organize more interesting structure with some property names, instead of plain arrays